mailbox/cassandra/src/test/resources/json/solveMessageInconsistencies.additionalInformation.json

```diff
@@ -0,0 +1,16 @@
+{
+  "type":"solve-message-inconsistencies",
+  "timestamp":"2007-12-03T10:15:30Z",
+  "processedImapUidEntries": 2,
+  "processedMessageIdEntries": 1,
+  "addedMessageIdEntries": 1,
+  "updatedMessageIdEntries": 0,
+  "removedMessageIdEntries": 1,
+  "fixedInconsistencies": [
+    "ComposedMessageId{mailboxId=CassandraId{id=551f0580-82fb-11ea-970e-f9c83d4cf8c2}, messageId=CassandraMessageId{uuid=d2bee791-7e63-11ea-883c-95b84008f979}, uid=MessageUid{uid=1}}",
```

Comment:
this serialization is not ok: we should leverage json format if we need to serialize several information